docs/how-to-guides/azure_resource_provision.json

```diff
@@ -380,9 +391,16 @@
               "value": "api/v1"
             },
             {
-            "condition": "[equals(parameters('registryBackend'),'Azure-SQL')]",
               "name": "CONNECTION_STR",
               "value": "[concat('Server=tcp:',reference(variables('sqlServerName')).fullyQualifiedDomainName,',1433;Initial Catalog=',variables('sqlDatabaseName'),';Persist Security Info=False;User ID=',reference(variables('sqlserverName')).administratorLogin,';Password=',parameters('sqlAdminPassword'),';MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;')]"
+            },
+            {
+              "name": "ENABLE_RBAC",
+              "value": "[if(equals(parameters('enableRBAC'),'Yes'),json('true'),json('null'))]"
+            },
+            {
+              "name": "PURVIEW_NAME",
+              "value": "[if(equals(parameters('registryBackend'),'Azure-Purview'),variables('purviewName'),json('null'))]"
             }
```

Comment:
When Purview is selected as registry provider, AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET are required, these is the service principal registry server uses to talk to purview.